src/CMakeLists.txt

```diff
@@ -659,7 +659,13 @@ if(ALICEVISION_HAVE_CUDA)
 
   set(CUDA_NVCC_FLAGS_DEBUG   "${CUDA_NVCC_FLAGS_DEBUG};-G;-g")
   # set(CUDA_NVCC_FLAGS_RELEASE "${CUDA_NVCC_FLAGS_RELEASE};-O3")
-  if(CUDA_VERSION VERSION_GREATER_EQUAL 11.1)
+  if(CUDA_VERSION_MAJOR VERSION_GREATER_EQUAL 12)
+    set(ALICEVISION_CUDA_CC_LIST_BASIC 50 52 60 61 62 70 72 75 80 86 87 89 90)
+  elseif(CUDA_VERSION VERSION_GREATER_EQUAL 11.8)
+    set(ALICEVISION_CUDA_CC_LIST_BASIC 35 50 52 60 61 62 70 72 75 80 86 87 89 90)
```

Comment:
Actually, since 11.6 everything < 60 has been deprecated, and we'll get a lot of noisy messages.
Also apparently 90 Is not supported in 11.8
https://arnon.dk/tag/nvcc-flags/